torat_client/rpcapi.go

```diff
@@ -74,9 +75,22 @@ func (a *API) LS(v shared.Void, r *shared.Dir) (err error) {
 	return
 }
 
-func (a *API) Ping(v shared.Void, r *string) error {
-	// TODO implement
-	*r = "Pong"
+func (a *API) Speedtest(v shared.Void, r *shared.Speedtest) error {
+	user, _ := speedtest.FetchUserInfo()
```

Comment:
Whats the other return value?
If its an error don't ignore it.

Reply:
I'm not sure what the other value returned is but that's how the official docs say to call it.
https://github.com/showwin/speedtest-go#api-usage